packages/gcp-functions/src/executors/deploy/deploy.impl.ts

```diff
@@ -45,7 +45,7 @@ export async function deployExecutor(
     triggerValue = null,
     triggerEvent = null,
     triggerLocation = null,
-    runtime = 'nodejs16',
+    runtime = 'nodejs18',
```

Comment:
Could you keep this one default at `nodejs16` as changing it to `nodejs18` will make it a breaking change. Or commit it with `BREAKING CHANGE: Updated to nodejs18 as recommended by Google` in the commit body.